<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Раскрашиватель фотографий</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/icon.png</normaloff>:/icons/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="colorize_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Раскрасить</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/icons/palette.svg</normaloff>:/icons/icons/palette.svg</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QScrollArea" name="source_scroll_area">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="source_scroll_area_widget_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>389</width>
         <height>516</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="source_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QScrollArea" name="result_scroll_area">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="result_scroll_area_widget_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>389</width>
         <height>516</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="result_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="save_action"/>
    <addaction name="save_as_action"/>
    <addaction name="close_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/file-image.svg</normaloff>:/icons/icons/file-image.svg</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="save_as_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/copy.svg</normaloff>:/icons/icons/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="close_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/times.svg</normaloff>:/icons/icons/times.svg</iconset>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="about_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/info-circle.svg</normaloff>:/icons/icons/info-circle.svg</iconset>
   </property>
   <property name="text">
    <string>О программе...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="save_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save.svg</normaloff>:/icons/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
